lib/engines/content_block_manager/features/create_object.feature

```diff
@@ -52,6 +52,18 @@ Feature: Create a content object
       | my email address | xxxxx           | Somewhere  | Ministry of Example |
     Then I should see a message that the "email_address" field is an invalid "email"
 
+  Scenario: GDS editor cancels the creation of an object
+    When I visit the Content Block Manager home page
+    And I click to create an object
+    When I click on the "email_address" schema
+    When I complete the form with the following fields:
+      | title            | email_address   | department | organisation |
+      | my email address | foo@example.com | Somewhere  | Ministry of Example |
+    And I click cancel
+    Then I am taken back to Content Block Manager home page
+    And no draft Content Block Edition has been created
```

Comment:
I'm diverting a little here from feature tests just testing changes that are visible to the user, but I'm not sure that just using integration tests would give us the coverage needed - I want to be sure that the deletion has happened when the user clicks this button

Reply:
Yeah, I think that's cool 👍